CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute.
+
+## Required tools
+
+- node.js
+- npm
+- on mac-os, you have to install xcode command line developer tools (run xcode-select --install)
+- gcloud
+- docker
+- the IDE of your choice
+
+
+## Project setup
+
+- run `npm install`
+- add default keys definitions
+  - `cp apps/fxc-front/src/app/keys.ts.dist apps/fxc-front/src/app/keys.ts`
+  - `cp libs/common/src/lib/keys.ts.dist libs/common/src/lib/keys.ts`
+
+### Simplistic configuration
+
+**redis server**
+- `cd docker; docker compose up -d redis`
+
+**pubsub**
+- `cd docker; docker compose up -d pubsub`
+
+**datastore**
+
+For the moment, it does not work with docker compose. But if you install the cloud-datastore-emulator, you will have a working configuration.
+
+***Installation***
+- `gcloud components install cloud-datastore-emulator`
+
+***run:***
+- `gcloud beta emulators datastore start`
```

Comment:
See https://cloud.google.com/sdk/gcloud/reference/beta/emulators/datastore

I think it's good to specify a `--data-dir` inside the project then it's easy to delete this folder to start with a fresh DB.

You can also use [`eval $(gcloud beta emulators datastore env-init --data-dir=DATA-DIR)`](https://cloud.google.com/sdk/gcloud/reference/beta/emulators/datastore/env-init) for setting the env vars

Reply:
ok. Documentation fixed